modules/nf-core/velocyto/tests/main.nf.test

```diff
@@ -0,0 +1,155 @@
+// nf-core modules test velocyto
+nextflow_process {
+    name 'Test Process VELOCYTO'
+    tag "modules"
+    tag "modules_nfcore"
+    tag "velocyto"
+    tag "samtools/sort"
+
+    script '../main.nf'
+    process 'VELOCYTO'
+
+    
+    
+    test('mus_musculus - barcodes and bam with setup') {
+        setup {
+            run("SAMTOOLS_SORT") {
+                script "../../samtools/sort/main.nf"
+                config "modules/nf-core/velocyto/tests/test.config"
+                
+                process {                                        
+
+                    """
+                    input[0] = [
+                        [ id:'Sample_X' ], // meta map
+                        file(params.test_data['mus_musculus']['velocity']['bam'], checkIfExists: true)
```

Comment:
Can you update the filepaths according to #5281 ? :)

Reply:
Thanks for review this!

Paths updated to new base modules_testdata_base_path. Using

```
params {
    modules_testdata_base_path = "https://raw.githubusercontent.com/tucano/test-datasets/modules/data/"
}
```

FOr testing and looks good (is my fork)

Reply:
Ah on the final PR it should look a bit like
```
file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/vcf/test.vcf', checkIfExists: true),
```